docs/inspec_and_friends.md

```diff
@@ -40,6 +40,14 @@ control "sshd-11" do
 end
 ```
 
+That said, InSpec is not RSpec. InSpec does not support arbitrary RSpec features. Some key differences:
```

Comment:
One key difference I might point out, since it comes up from time-to-time, is that InSpec doesn't support `shared_examples`... instead, we recommend creating a custom resource that performs the repetitious tests as needed.

Reply:
> **arbitrary** - based on random choice or personal whim, rather than any reason or system.
I would consider a different adjective. The text that you use below to describe RSpec's focus as a tool in supporting software engineers feels like it could help describe the goal of their project versus the compliance focus of this project.